packages/website/scripts/compileDocsForLLM.js

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env node
+const { glob } = require("glob");
+const fs = require("fs");
+
+const readFile = (fileName) => {
+  return fs.readFileSync(fileName, "utf-8");
+};
+
+const documentationBundlePage = async () => {
+  const targetFilename = "./public/llms/documentationBundle.txt";
+
+  const header = `# Squiggle Documentation, One Page
+This file is auto-generated from the documentation files in the Squiggle repository. It includes our Peggy Grammar. It is meant to be given to an LLM. It is not meant to be read by humans.
+--- \n\n
+`;
+
+  const getGrammarContent = async () => {
+    const grammarFiles = await glob("../squiggle-lang/**/*.peggy");
```

Comment:
```suggestion
    const grammarFiles = await glob("../squiggle-lang/src/**/*.peggy");
```

Otherwise it'd look through the entire `node_modules/`, can be slow and there's a risk that `node_modules/peggy/` contains example files (it doesn't right now but that can change).